fintrack/src/components/UserSignOut.js

```diff
@@ -0,0 +1,18 @@
+import { useContext, useEffect } from 'react'
+import UserContext from '../context/UserContext'
+import { Navigate } from 'react-router-dom'
+
+
+
+const UserSignOut = () => {
+    const { actions } = useContext(UserContext)
+
+    // takes a function to run when the component is rendered
+    useEffect(() => actions.signOut())
```

Comment:
```suggestion
  actions.signOut()
```

unnecessary call to `useEffect`!